palantir-java-format/src/main/java/com/palantir/javaformat/java/JavaOutput.java

```diff
@@ -193,11 +172,6 @@ void flush() {
 
     // The following methods can be used after the Output has been built.
 
-    @Override
-    public CommentsHelper getCommentsHelper() {
```

Comment:
This was not used at all in this class, was only read from the Formatter.format method so it was essentially used to bundle two values into one.